### PR TITLE
lint: remove identified duplicate includes

### PR DESCRIPTION
**Pull request by barton2526**
~~Duplicate include(s) in src/qt/bitcoin.cpp:~~
~~#include <QtPlugin>~~

Duplicate include(s) in src/qt/qtipcserver.cpp:
#include <boost/version.hpp>

Duplicate include(s) in src/qt/sendcoinsdialog.cpp:
#include "addressbookpage.h"

^---- failure generated from test/lint/lint-includes.sh

### CODE REVIEW

File: src/qt/bitcoin.cpp [hunk 1]
```diff
@@ -38,7 +38,6 @@
 #if defined(BITCOIN_NEED_QT_PLUGINS) && !defined(_BITCOIN_QT_PLUGINS_INCLUDED)
 #define _BITCOIN_QT_PLUGINS_INCLUDED
```
Comment (jamescowens) [inline src/qt/bitcoin.cpp]: I think this is a false positive.

Comment (barton2526) [reply src/qt/bitcoin.cpp]: but line 41 and 50?

Comment (jamescowens) [reply src/qt/bitcoin.cpp]: If you look closely each of them are nested inside #if define statements with different matching conditions.

Comment (barton2526) [reply src/qt/bitcoin.cpp]: ah, I missed that entirely.  I'll revert that one

**Review by jamescowens — APPROVED** [review]

**Pull request merged**